client/post-editor/post-editor.jsx

```diff
@@ -1396,26 +1395,21 @@ const enhance = flow(
 				nestedSidebarTarget: getNestedSidebarTarget( state ),
 			};
 		},
-		dispatch => {
-			return bindActionCreators(
-				{
-					setEditorLastDraft,
-					resetEditorLastDraft,
-					receivePost,
-					editPost,
-					savePostSuccess,
-					setEditorModePreference: savePreference.bind( null, 'editor-mode' ),
-					setEditorSidebar: savePreference.bind( null, 'editor-sidebar' ),
-					setLayoutFocus,
-					setNextLayoutFocus,
-					saveConfirmationSidebarPreference,
-					recordTracksEvent,
-					setNestedSidebar,
-					closeEditorSidebar,
-					openEditorSidebar,
-				},
-				dispatch
-			);
+		{
+			setEditorLastDraft,
+			resetEditorLastDraft,
+			receivePost,
+			editPost,
+			savePostSuccess,
+			setEditorModePreference: partial( savePreference, 'editor-mode' ),
```

Comment:
Do we really need to import `partial` just for this couple of dispatches?

Reply:
I do prefer the use of `lodash` for utilities but I'd be happy to revert to `.bind`, the issue would still be solved :)

Reply:
Yes, it's a super minor nitpick, just because it stuck out to me to introduce a new utility when trimming out stuff. 😄 
No big deal either way though!

Reply:
I like `partial` here for the clarity, we're already bringing in other lodash utils in this file.